The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9907822501/job/27372316324) [(plain)](https://github.com/rust-lang/rust/commit/b203aa93b5769ad1ea822f4a62752f330a9996b7/checks/27372316324/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Getting action download info
Download action repository 'msys2/setup-msys2@v2.22.0' (SHA:cc11e9188b693c2b100158c3322424c4cc1dadea)
Download action repository 'actions/checkout@v4' (SHA:692973e3d937129bcbf40652eb9f2f61becf3332)
Download action repository 'actions/upload-artifact@v4' (SHA:0b2256b8c012f0828dc542b3febcab082c67f72b)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
COPY scripts/sccache.sh /scripts/
RUN sh /scripts/sccache.sh

COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt \
    && pip3 install virtualenv
COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---

#12 [5/8] COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
#12 DONE 0.0s

#13 [6/8] RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
#13 0.462   Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
#13 0.475 Collecting boolean-py==4.0
#13 0.478   Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
#13 0.491 Collecting chardet==5.1.0
---
#13 3.694 Building wheels for collected packages: reuse
#13 3.695   Building wheel for reuse (pyproject.toml): started
#13 4.043   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 4.044   Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=181117 sha256=f5f58750481f69515c2c0d1d503daf565e2565c370d07fc6aeb95fe3498b4269
#13 4.044   Stored in directory: /tmp/pip-ephem-wheel-cache-7287puu8/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
#13 4.047 Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
#13 4.070   Attempting uninstall: setuptools
#13 4.071     Found existing installation: setuptools 59.6.0
#13 4.072     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#13 4.072     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#13 4.073     Can't uninstall 'setuptools'. No files were found to uninstall.
#13 4.776 Successfully installed binaryornot-0.4.4 boolean-py-4.0 chardet-5.1.0 jinja2-3.1.2 license-expression-30.0.0 markupsafe-2.1.1 python-debian-0.1.49 reuse-1.1.0 setuptools-66.0.0
#13 4.776 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 5.342 Collecting virtualenv
#13 5.383   Downloading virtualenv-20.26.3-py3-none-any.whl (5.7 MB)
#13 5.458      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.7/5.7 MB 77.6 MB/s eta 0:00:00
#13 5.509 Collecting platformdirs<5,>=3.9.1
#13 5.512   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 5.546 Collecting filelock<4,>=3.12.2
#13 5.550   Downloading filelock-3.15.4-py3-none-any.whl (16 kB)
#13 5.567 Collecting distlib<1,>=0.3.7
#13 5.570   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 5.578      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 87.6 MB/s eta 0:00:00
#13 5.668 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 5.873 Successfully installed distlib-0.3.8 filelock-3.15.4 platformdirs-4.2.2 virtualenv-20.26.3
#13 DONE 6.0s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      217024 kB
DirectMap2M:     9220096 kB
DirectMap1G:     9437184 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/b286722878e18db29a7fbe672be7c4d3b02e8e4d/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-b286722878e18db29a7fbe672be7c4d3b02e8e4d-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 29.40s
##[endgroup]
fmt check
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/src/common.rs at line 58:
 
 impl<'gcc, 'tcx> ConstMethods<'tcx> for CodegenCx<'gcc, 'tcx> {
     fn const_null(&self, typ: Type<'gcc>) -> RValue<'gcc> {
-        if type_is_pointer(typ) {
-            self.context.new_null(typ)
-            self.const_int(typ, 0)
-        }
-        }
+        if type_is_pointer(typ) { self.context.new_null(typ) } else { self.const_int(typ, 0) }
 
 
     fn const_undef(&self, typ: Type<'gcc>) -> RValue<'gcc> {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 54:
                     config.gcc_path = Some(value.as_str().to_string())
                 }
                 ("gcc-path", _) => {
-                    return failed_config_parsing(config_file, "Expected a string for `gcc-path`")
+                    return failed_config_parsing(config_file, "Expected a string for `gcc-path`");
                 }
                 ("download-gccjit", TomlValue::Boolean(value)) => {
                     config.download_gccjit = Some(*value)
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 63:
                     return failed_config_parsing(
                         "Expected a boolean for `download-gccjit`",
-                    )
+                    );
                 }
                 }
                 _ => return failed_config_parsing(config_file, &format!("Unknown key `{}`", key)),
             }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 73:
                 return failed_config_parsing(
                     config_file,
                     "At least one of `gcc-path` or `download-gccjit` value must be set",
+                );
             }
             (Some(_), Some(true)) => {
                 println!(
                 println!(
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 144:
                     return Err(
                     return Err(
                         "Expected a value after `--target-triple`, found nothing".to_string()
+                    );
                 }
             },
             "--out-dir" => match args.next() {
             "--out-dir" => match args.next() {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 158:
                     self.config_file = Some(arg.to_string());
                 _ => {
                 _ => {
-                    return Err("Expected a value after `--config-file`, found nothing".to_string())
+                    return Err("Expected a value after `--config-file`, found nothing".to_string());
             },
             "--release-sysroot" => self.sysroot_release_channel = true,
             "--release-sysroot" => self.sysroot_release_channel = true,
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 169:
                     self.cg_gcc_path = Some(arg.into());
                 _ => {
                 _ => {
-                    return Err("Expected a value after `--cg_gcc-path`, found nothing".to_string())
+                    return Err("Expected a value after `--cg_gcc-path`, found nothing".to_string());
             },
             },
             "--use-backend" => match args.next() {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs at line 277:
         self.gcc_path = match gcc_path {
             Some(path) => path,
             None => {
-                return Err(format!("missing `gcc-path` value from `{}`", config_file.display(),))
+                return Err(format!("missing `gcc-path` value from `{}`", config_file.display(),));
         };
         Ok(())
         Ok(())
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/src/abi.rs at line 26:
             false
         };
-        if on_stack {
-        if on_stack {
-            param.to_lvalue().get_address(None)
-            param.to_rvalue()
-        }
-        }
+        if on_stack { param.to_lvalue().get_address(None) } else { param.to_rvalue() }
 }
 
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/test.rs at line 109:
                         test_arg.flags.extend_from_slice(&["--features".into(), feature]);
                     _ => {
                     _ => {
-                        return Err("Expected an argument after `--features`, found nothing".into())
+                        return Err("Expected an argument after `--features`, found nothing".into());
                 },
                 "--use-system-gcc" => {
                 "--use-system-gcc" => {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/test.rs at line 458:
     .map_err(|error| format!("Failed to retrieve cargo path: {:?}", error))
     .and_then(|cargo| {
         let cargo = cargo.trim().to_owned();
-        if cargo.is_empty() {
-            Err(format!("`cargo` path is empty"))
-            Ok(cargo)
-        }
-        }
+        if cargo.is_empty() { Err(format!("`cargo` path is empty")) } else { Ok(cargo) }
     let rustc = String::from_utf8(
     let rustc = String::from_utf8(
         run_command_with_env(&[&"rustup", &toolchain, &"which", &"rustc"], rust_dir, Some(env))?
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/test.rs at line 471:
     .map_err(|error| format!("Failed to retrieve rustc path: {:?}", error))
     .and_then(|rustc| {
         let rustc = rustc.trim().to_owned();
-        if rustc.is_empty() {
-            Err(format!("`rustc` path is empty"))
-            Ok(rustc)
-        }
-        }
+        if rustc.is_empty() { Err(format!("`rustc` path is empty")) } else { Ok(rustc) }
     let llvm_filecheck = match run_command_with_env(
         &[
         &[
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/src/builder.rs at line 1043:
                 let llty = place.layout.scalar_pair_element_gcc_type(self, i);
                 let load = self.load(llty, llptr, align);
                 scalar_load_metadata(self, load, scalar);
-                if scalar.is_bool() {
-                    self.trunc(load, self.type_i1())
-                    load
-                }
-                }
+                if scalar.is_bool() { self.trunc(load, self.type_i1()) } else { load }
 
             OperandValue::Pair(
             OperandValue::Pair(
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/src/builder.rs at line 1795:
         // This already happens today with u128::MAX = 2^128 - 1 > f32::MAX.
         let int_max = |signed: bool, int_width: u64| -> u128 {
             let shift_amount = 128 - int_width;
-            if signed {
-                i128::MAX as u128 >> shift_amount
-                u128::MAX >> shift_amount
-            }
-            }
+            if signed { i128::MAX as u128 >> shift_amount } else { u128::MAX >> shift_amount }
         };
         let int_min = |signed: bool, int_width: u64| -> i128 {
-            if signed {
-                i128::MIN >> (128 - int_width)
-                0
-            }
-            }
+            if signed { i128::MIN >> (128 - int_width) } else { 0 }
 
 
         let compute_clamp_bounds_single = |signed: bool, int_width: u64| -> (u128, u128) {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/utils.rs at line 175:
 pub fn get_os_name() -> Result<String, String> {
     let output = run_command(&[&"uname"], None)?;
     let name = std::str::from_utf8(&output.stdout).unwrap_or("").trim().to_string();
-    if !name.is_empty() {
-        Ok(name)
-    } else {
-        Err("Failed to retrieve the OS name".to_string())
-    }
+    if !name.is_empty() { Ok(name) } else { Err("Failed to retrieve the OS name".to_string()) }
 
 #[derive(Default, PartialEq)]
 #[derive(Default, PartialEq)]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/src/asm.rs at line 858:
         InlineAsmRegClass::AArch64(AArch64InlineAsmRegClass::reg) => modifier,
         InlineAsmRegClass::AArch64(AArch64InlineAsmRegClass::vreg)
         | InlineAsmRegClass::AArch64(AArch64InlineAsmRegClass::vreg_low16) => {
-            if modifier == Some('v') {
-            } else {
-                modifier
-            }
-            }
+            if modifier == Some('v') { None } else { modifier }
         }
         InlineAsmRegClass::AArch64(AArch64InlineAsmRegClass::preg) => {
             unreachable!("clobber-only")
error: expected item, found keyword `match`
  |
3 | match name {
  | ^^^^^ expected item
  |
  |
  = note: for a full list of items that can appear in modules, see <https://doc.rust-lang.org/reference/items.html>

##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mod_bench.rs at line 3:
 #![allow(internal_features)]
 #[link(name = "c")]
-extern {}
+extern "C" {}
 
 
 #[panic_handler]
 fn panic_handler(_: &core::panic::PanicInfo<'_>) -> ! {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mod_bench.rs at line 10:
 }
 
 
-#[lang="eh_personality"]
-fn eh_personality(){}
+#[lang = "eh_personality"]
+fn eh_personality() {}
 // Required for rustc_codegen_llvm
 #[no_mangle]
 #[no_mangle]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 1:
 // Adapted from https://github.com/sunfishcode/mir2cranelift/blob/master/rust-examples/nocore-hello-world.rs
 #![feature(
-    no_core, unboxed_closures, start, lang_items, never_type, linkage,
-    extern_types, thread_local, raw_ref_op
+    no_core,
---
+    raw_ref_op
 )]
 #![no_core]
 #![allow(dead_code, internal_features, non_camel_case_types)]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 9:
 extern crate mini_core;
 
-use mini_core::*;
 use mini_core::libc::*;
 use mini_core::libc::*;
+use mini_core::*;
 
 unsafe extern "C" fn my_puts(s: *const u8) {
     puts(s);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 88:
     _sigpipe: u8,
 ) -> isize {
     if argc == 3 {
-        unsafe { puts(*argv); }
-        unsafe { puts(*((argv as usize + intrinsics::size_of::<*const u8>()) as *const *const u8)); }
-        unsafe { puts(*((argv as usize + 2 * intrinsics::size_of::<*const u8>()) as *const *const u8)); }
+        unsafe {
+            puts(*argv);
+        unsafe {
+        unsafe {
+            puts(*((argv as usize + intrinsics::size_of::<*const u8>()) as *const *const u8));
+        unsafe {
+        unsafe {
+            puts(*((argv as usize + 2 * intrinsics::size_of::<*const u8>()) as *const *const u8));
     }
 
     main().report();
     main().report();
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 99:
 
 static mut NUM: u8 = 6 * 7;
 
-static NUM_REF: &'static u8 = unsafe { &* &raw const NUM };
+static NUM_REF: &'static u8 = unsafe { &*&raw const NUM };
 macro_rules! assert {
     ($e:expr) => {
     ($e:expr) => {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 106:
         if !$e {
-            panic(stringify!(! $e));
+            panic(stringify!(!$e));
     };
 }
 }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 114:
         if $l != $r {
             panic(stringify!($l != $r));
-    }
+    };
 }
 
 
 struct Unique<T: ?Sized> {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 142:
 
 fn main() {
-    take_unique(Unique {
-        pointer: 0 as *const (),
-        pointer: 0 as *const (),
-        _marker: PhantomData,
-    });
+    take_unique(Unique { pointer: 0 as *const (), _marker: PhantomData });
     take_f32(0.1);
 
     //call_return_u128_pair();
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 188:
         let a: &dyn SomeTrait = &"abc\0";
         a.object_safe();
 
-        #[cfg(target_arch="x86_64")]
+        #[cfg(target_arch = "x86_64")]
         assert_eq!(intrinsics::size_of_val(a) as u8, 16);
-        #[cfg(target_arch="m68k")]
+        #[cfg(target_arch = "m68k")]
         assert_eq!(intrinsics::size_of_val(a) as u8, 8);
         assert_eq!(intrinsics::size_of_val(&0u32) as u8, 4);
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 197:
         assert_eq!(intrinsics::min_align_of::<u16>() as u8, 2);
-        assert_eq!(intrinsics::min_align_of_val(&a) as u8, intrinsics::min_align_of::<&str>() as u8);
+        assert_eq!(
+            intrinsics::min_align_of_val(&a) as u8,
+            intrinsics::min_align_of::<&str>() as u8
 
 
         assert!(!intrinsics::needs_drop::<u8>());
         assert!(!intrinsics::needs_drop::<[u8]>());
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 202:
         assert!(intrinsics::needs_drop::<NoisyDrop>());
         assert!(intrinsics::needs_drop::<NoisyDropUnsized>());
-        Unique {
-            pointer: 0 as *const &str,
-            _marker: PhantomData,
-            _marker: PhantomData,
-        } as Unique<dyn SomeTrait>;
+        Unique { pointer: 0 as *const &str, _marker: PhantomData } as Unique<dyn SomeTrait>;
 
         struct MyDst<T: ?Sized>(T);
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 231:
     }
 
 
-    let _ = Box::new(NoisyDrop {
-        text: "Boxed outer got dropped!\0",
-        inner: NoisyDropInner,
-    }) as Box<dyn SomeTrait>;
+    let _ = Box::new(NoisyDrop { text: "Boxed outer got dropped!\0", inner: NoisyDropInner })
+        as Box<dyn SomeTrait>;
 
     const FUNC_REF: Option<fn()> = Some(main);
     #[allow(unreachable_code)]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 241:
     match FUNC_REF {
-        Some(_) => {},
+        Some(_) => {}
         None => assert!(false),
 
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 246:
     match Ordering::Less {
-        Ordering::Less => {},
+        Ordering::Less => {}
         _ => assert!(false),
 
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 258:
 
     assert_eq!(((|()| 42u8) as fn(()) -> u8)(()), 42);
-    extern {
+    extern "C" {
         #[linkage = "weak"]
         #[linkage = "weak"]
         static ABC: *const u8;
     }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 265:
     {
-        extern {
+        extern "C" {
             #[linkage = "weak"]
             #[linkage = "weak"]
             static ABC: *const u8;
         }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 328:
         native: *mut pthread_t,
         attr: *const pthread_attr_t,
         f: extern "C" fn(_: *mut c_void) -> *mut c_void,
+        value: *mut c_void,
     ) -> c_int;
 
-    fn pthread_join(
-    fn pthread_join(
-        native: pthread_t,
-        value: *mut *mut c_void
-    ) -> c_int;
+    fn pthread_join(native: pthread_t, value: *mut *mut c_void) -> c_int;
 
 #[thread_local]
 #[thread_local]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 343:
 
 #[cfg(not(jit))]
 extern "C" fn mutate_tls(_: *mut c_void) -> *mut c_void {
-    unsafe { TLS = 0; }
+        TLS = 0;
+    }
     0 as *mut c_void
 }
 }
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 393:
 pub enum E2<X> {
     V1 { f: bool },
 
-    /*_00*/ _01(X), _02(X), _03(X), _04(X), _05(X), _06(X), _07(X),
-    _08(X), _09(X), _0A(X), _0B(X), _0C(X), _0D(X), _0E(X), _0F(X),
-    _10(X), _11(X), _12(X), _13(X), _14(X), _15(X), _16(X), _17(X),
-    _18(X), _19(X), _1A(X), _1B(X), _1C(X), _1D(X), _1E(X), _1F(X),
-    _20(X), _21(X), _22(X), _23(X), _24(X), _25(X), _26(X), _27(X),
-    _28(X), _29(X), _2A(X), _2B(X), _2C(X), _2D(X), _2E(X), _2F(X),
-    _30(X), _31(X), _32(X), _33(X), _34(X), _35(X), _36(X), _37(X),
-    _38(X), _39(X), _3A(X), _3B(X), _3C(X), _3D(X), _3E(X), _3F(X),
-    _40(X), _41(X), _42(X), _43(X), _44(X), _45(X), _46(X), _47(X),
-    _48(X), _49(X), _4A(X), _4B(X), _4C(X), _4D(X), _4E(X), _4F(X),
-    _50(X), _51(X), _52(X), _53(X), _54(X), _55(X), _56(X), _57(X),
-    _58(X), _59(X), _5A(X), _5B(X), _5C(X), _5D(X), _5E(X), _5F(X),
-    _60(X), _61(X), _62(X), _63(X), _64(X), _65(X), _66(X), _67(X),
-    _68(X), _69(X), _6A(X), _6B(X), _6C(X), _6D(X), _6E(X), _6F(X),
-    _70(X), _71(X), _72(X), _73(X), _74(X), _75(X), _76(X), _77(X),
-    _78(X), _79(X), _7A(X), _7B(X), _7C(X), _7D(X), _7E(X), _7F(X),
-    _80(X), _81(X), _82(X), _83(X), _84(X), _85(X), _86(X), _87(X),
-    _88(X), _89(X), _8A(X), _8B(X), _8C(X), _8D(X), _8E(X), _8F(X),
-    _90(X), _91(X), _92(X), _93(X), _94(X), _95(X), _96(X), _97(X),
-    _98(X), _99(X), _9A(X), _9B(X), _9C(X), _9D(X), _9E(X), _9F(X),
-    _A0(X), _A1(X), _A2(X), _A3(X), _A4(X), _A5(X), _A6(X), _A7(X),
-    _A8(X), _A9(X), _AA(X), _AB(X), _AC(X), _AD(X), _AE(X), _AF(X),
-    _B0(X), _B1(X), _B2(X), _B3(X), _B4(X), _B5(X), _B6(X), _B7(X),
-    _B8(X), _B9(X), _BA(X), _BB(X), _BC(X), _BD(X), _BE(X), _BF(X),
-    _C0(X), _C1(X), _C2(X), _C3(X), _C4(X), _C5(X), _C6(X), _C7(X),
-    _C8(X), _C9(X), _CA(X), _CB(X), _CC(X), _CD(X), _CE(X), _CF(X),
-    _D0(X), _D1(X), _D2(X), _D3(X), _D4(X), _D5(X), _D6(X), _D7(X),
-    _D8(X), _D9(X), _DA(X), _DB(X), _DC(X), _DD(X), _DE(X), _DF(X),
-    _E0(X), _E1(X), _E2(X), _E3(X), _E4(X), _E5(X), _E6(X), _E7(X),
-    _E8(X), _E9(X), _EA(X), _EB(X), _EC(X), _ED(X), _EE(X), _EF(X),
-    _F0(X), _F1(X), _F2(X), _F3(X), _F4(X), _F5(X), _F6(X), _F7(X),
-    _F8(X), _F9(X), _FA(X), _FB(X), _FC(X), _FD(X), _FE(X), _FF(X),
+    /*_00*/ _01(X),
+    _02(X),
+    _03(X),
+    _04(X),
+    _05(X),
+    _06(X),
+    _07(X),
+    _08(X),
+    _09(X),
+    _0A(X),
+    _0B(X),
+    _0C(X),
+    _0D(X),
+    _0E(X),
+    _0F(X),
+    _10(X),
+    _11(X),
+    _12(X),
+    _13(X),
+    _14(X),
+    _15(X),
+    _16(X),
+    _17(X),
+    _18(X),
+    _19(X),
+    _1A(X),
+    _1B(X),
+    _1C(X),
+    _1D(X),
+    _1E(X),
+    _1F(X),
+    _20(X),
+    _21(X),
+    _22(X),
+    _23(X),
+    _24(X),
+    _25(X),
+    _26(X),
+    _27(X),
+    _28(X),
+    _29(X),
+    _2A(X),
+    _2B(X),
+    _2C(X),
+    _2D(X),
+    _2E(X),
+    _2F(X),
+    _30(X),
+    _31(X),
+    _32(X),
+    _33(X),
+    _34(X),
+    _35(X),
+    _36(X),
+    _37(X),
+    _38(X),
+    _39(X),
+    _3A(X),
+    _3B(X),
+    _3C(X),
+    _3D(X),
+    _3E(X),
+    _3F(X),
+    _40(X),
+    _41(X),
+    _42(X),
+    _43(X),
+    _44(X),
+    _45(X),
+    _46(X),
+    _47(X),
+    _48(X),
+    _49(X),
+    _4A(X),
+    _4B(X),
+    _4C(X),
+    _4D(X),
+    _4E(X),
+    _4F(X),
+    _50(X),
+    _51(X),
+    _52(X),
+    _53(X),
+    _54(X),
+    _55(X),
+    _56(X),
+    _57(X),
+    _58(X),
+    _59(X),
+    _5A(X),
+    _5B(X),
+    _5C(X),
+    _5D(X),
+    _5E(X),
+    _5F(X),
+    _60(X),
+    _61(X),
+    _62(X),
+    _63(X),
+    _64(X),
+    _65(X),
+    _66(X),
+    _67(X),
+    _68(X),
+    _69(X),
+    _6A(X),
+    _6B(X),
+    _6C(X),
+    _6D(X),
+    _6E(X),
+    _6F(X),
+    _70(X),
+    _71(X),
+    _72(X),
+    _73(X),
+    _74(X),
+    _75(X),
+    _76(X),
+    _77(X),
+    _78(X),
+    _79(X),
+    _7A(X),
+    _7B(X),
+    _7C(X),
+    _7D(X),
+    _7E(X),
+    _7F(X),
+    _80(X),
+    _81(X),
+    _82(X),
+    _83(X),
+    _84(X),
+    _85(X),
+    _86(X),
+    _87(X),
+    _88(X),
+    _89(X),
+    _8A(X),
+    _8B(X),
+    _8C(X),
+    _8D(X),
+    _8E(X),
+    _8F(X),
+    _90(X),
+    _91(X),
+    _92(X),
+    _93(X),
+    _94(X),
+    _95(X),
+    _96(X),
+    _97(X),
+    _98(X),
+    _99(X),
+    _9A(X),
+    _9B(X),
+    _9C(X),
+    _9D(X),
+    _9E(X),
+    _9F(X),
+    _A0(X),
+    _A1(X),
+    _A2(X),
+    _A3(X),
+    _A4(X),
+    _A5(X),
+    _A6(X),
+    _A7(X),
+    _A8(X),
+    _A9(X),
+    _AA(X),
+    _AB(X),
+    _AC(X),
+    _AD(X),
+    _AE(X),
+    _AF(X),
+    _B0(X),
+    _B1(X),
+    _B2(X),
+    _B3(X),
+    _B4(X),
+    _B5(X),
+    _B6(X),
+    _B7(X),
+    _B8(X),
+    _B9(X),
+    _BA(X),
+    _BB(X),
+    _BC(X),
+    _BD(X),
+    _BE(X),
+    _BF(X),
+    _C0(X),
+    _C1(X),
+    _C2(X),
+    _C3(X),
+    _C4(X),
+    _C5(X),
+    _C6(X),
+    _C7(X),
+    _C8(X),
+    _C9(X),
+    _CA(X),
+    _CB(X),
+    _CC(X),
+    _CD(X),
+    _CE(X),
+    _CF(X),
+    _D0(X),
+    _D1(X),
+    _D2(X),
+    _D3(X),
+    _D4(X),
+    _D5(X),
+    _D6(X),
+    _D7(X),
+    _D8(X),
+    _D9(X),
+    _DA(X),
+    _DB(X),
+    _DC(X),
+    _DD(X),
+    _DE(X),
+    _DF(X),
+    _E0(X),
+    _E1(X),
+    _E2(X),
+    _E3(X),
+    _E4(X),
+    _E5(X),
+    _E6(X),
+    _E7(X),
+    _E8(X),
+    _E9(X),
+    _EA(X),
+    _EB(X),
+    _EC(X),
+    _ED(X),
+    _EE(X),
+    _EF(X),
+    _F0(X),
+    _F1(X),
+    _F2(X),
+    _F3(X),
+    _F4(X),
+    _F5(X),
+    _F6(X),
+    _F7(X),
+    _F8(X),
+    _F9(X),
+    _FA(X),
+    _FB(X),
+    _FC(X),
+    _FD(X),
+    _FE(X),
+    _FF(X),
     V3,
     V4,
     V4,
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core_hello_world.rs at line 431:
 
-fn check_niche_behavior () {
-fn check_niche_behavior () {
+fn check_niche_behavior() {
     if let E1::V2 { .. } = (E1::V1 { f: true }) {
     }
     }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/alloc_system.rs at line 6:
 // The minimum alignment guaranteed by the architecture. This value is used to
 // add fast paths for low alignment values.
 // add fast paths for low alignment values.
-#[cfg(any(target_arch = "x86",
-              target_arch = "arm",
-              target_arch = "m68k",
-              target_arch = "mips",
-              target_arch = "mips32r6",
-              target_arch = "powerpc",
-              target_arch = "csky",
-              target_arch = "powerpc64"))]
+#[cfg(any(
+    target_arch = "x86",
+    target_arch = "arm",
+    target_arch = "m68k",
+    target_arch = "mips",
+    target_arch = "mips32r6",
+    target_arch = "powerpc",
+    target_arch = "csky",
+))]
 const MIN_ALIGN: usize = 8;
 const MIN_ALIGN: usize = 8;
---
+    )));
     assert_eq!(mask1, 1);
 }
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 142:
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_slli_si128() {
     #[rustfmt::skip]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 167:
     assert_eq_m128i(r, _mm_set1_epi8(0));
 
-
-
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_movemask_epi8() {
     #[rustfmt::skip]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 183:
     assert_eq!(r, 0b10100100_00100101);
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "avx2")]
 unsafe fn test_mm256_movemask_epi8() {
     let a = _mm256_set1_epi8(-1);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 193:
     assert_eq!(r, e);
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_add_epi8() {
     let a = _mm_setr_epi8(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 210:
     assert_eq_m128i(r, e);
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_add_pd() {
     let a = _mm_setr_pd(1.0, 2.0);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 220:
     assert_eq_m128d(r, _mm_setr_pd(6.0, 12.0));
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 fn assert_eq_m128i(x: std::arch::x86_64::__m128i, y: std::arch::x86_64::__m128i) {
     unsafe {
         assert_eq!(std::mem::transmute::<_, [u8; 16]>(x), std::mem::transmute::<_, [u8; 16]>(y));
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 228:
 }
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 pub unsafe fn assert_eq_m128d(a: __m128d, b: __m128d) {
     if _mm_movemask_pd(_mm_cmpeq_pd(a, b)) != 0b11 {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 237:
 }
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_cvtsi128_si64() {
     let r = _mm_cvtsi128_si64(std::mem::transmute::<[i64; 2], _>([5, 0]));
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 245:
     assert_eq!(r, 5);
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse4.1")]
 unsafe fn test_mm_cvtepi8_epi16() {
     let a = _mm_set1_epi8(10);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 259:
     assert_eq_m128i(r, e);
 
 
-#[cfg(feature="master")]
-#[cfg(target_arch="x86_64")]
+#[cfg(feature = "master")]
+#[cfg(target_arch = "x86_64")]
 #[target_feature(enable = "sse4.1")]
 unsafe fn test_mm_extract_epi8() {
     #[rustfmt::skip]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 274:
     assert_eq!(r2, 3);
 
 
-#[cfg(all(feature="master", target_arch = "x86_64"))]
+#[cfg(all(feature = "master", target_arch = "x86_64"))]
 #[target_feature(enable = "sse2")]
 unsafe fn test_mm_insert_epi16() {
     let a = _mm_setr_epi16(0, 1, 2, 3, 4, 5, 6, 7);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/std_example.rs at line 286:
 #[derive(PartialEq)]
 enum LoopState {
     Continue(()),
-    Break(())
+    Break(()),
 
 pub enum Instruction {
 pub enum Instruction {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 1:
-    no_core, lang_items, intrinsics, unboxed_closures, type_ascription, extern_types,
-    decl_macro, rustc_attrs, transparent_unions, auto_traits, freeze_impls,
+    no_core,
+    lang_items,
---
+    auto_traits,
+    freeze_impls,
     thread_local
 )]
 #![no_core]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 35:
 pub trait DispatchFromDyn<T> {}
 
 // &T -> &U
-impl<'a, T: ?Sized+Unsize<U>, U: ?Sized> DispatchFromDyn<&'a U> for &'a T {}
+impl<'a, T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<&'a U> for &'a T {}
 // &mut T -> &mut U
-impl<'a, T: ?Sized+Unsize<U>, U: ?Sized> DispatchFromDyn<&'a mut U> for &'a mut T {}
+impl<'a, T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<&'a mut U> for &'a mut T {}
 // *const T -> *const U
-impl<T: ?Sized+Unsize<U>, U: ?Sized> DispatchFromDyn<*const U> for *const T {}
+impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<*const U> for *const T {}
 // *mut T -> *mut U
-impl<T: ?Sized+Unsize<U>, U: ?Sized> DispatchFromDyn<*mut U> for *mut T {}
+impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<*mut U> for *mut T {}
 impl<T: ?Sized + Unsize<U>, U: ?Sized> DispatchFromDyn<Box<U, ()>> for Box<T, ()> {}
 #[lang = "receiver"]
 #[lang = "receiver"]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 281:
 }
 
-
-
 impl PartialEq for u64 {
     fn eq(&self, other: &u64) -> bool {
         (*self) == (*other)
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 468:
 #[track_caller]
 fn panic_bounds_check(index: usize, len: usize) -> ! {
     unsafe {
-        libc::printf("index out of bounds: the len is %d but the index is %d\n\0" as *const str as *const i8, len, index);
+        libc::printf(
+            "index out of bounds: the len is %d but the index is %d\n\0" as *const str as *const i8,
+            len,
+        );
         intrinsics::abort();
     }
 }
 }
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 496:
     fn deref(&self) -> &Self::Target;
 
-pub trait Allocator {
-}
+pub trait Allocator {}
+pub trait Allocator {}
 
 impl Allocator for () {}
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 639:
 }
 
-extern {
+extern "C" {
+extern "C" {
     type VaListImpl;
 }
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 649:
 #[rustc_builtin_macro]
 #[rustc_builtin_macro]
 #[rustc_macro_transparency = "semitransparent"]
-pub macro stringify($($t:tt)*) { /* compiler built-in */ }
+pub macro stringify($($t:tt)*) {
+}
 
 #[rustc_builtin_macro]
 #[rustc_builtin_macro]
 #[rustc_macro_transparency = "semitransparent"]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 656:
-pub macro file() { /* compiler built-in */ }
+pub macro file() {
+}
 
 #[rustc_builtin_macro]
 #[rustc_builtin_macro]
 #[rustc_macro_transparency = "semitransparent"]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 660:
-pub macro line() { /* compiler built-in */ }
+pub macro line() {
+}
 
 #[rustc_builtin_macro]
 #[rustc_builtin_macro]
 #[rustc_macro_transparency = "semitransparent"]
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/mini_core.rs at line 664:
-pub macro cfg() { /* compiler built-in */ }
+pub macro cfg() {
+}
 
 
 pub static A_STATIC: u8 = 42;
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/arbitrary_self_types_pointers_and_wrappers.rs at line 5:
 #![allow(internal_features)]
 use std::{
 use std::{
-    ops::{Deref, CoerceUnsized, DispatchFromDyn},
     marker::Unsize,
+    ops::{CoerceUnsized, Deref, DispatchFromDyn},
 
 
 struct Ptr<T: ?Sized>(Box<T>);
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/arbitrary_self_types_pointers_and_wrappers.rs at line 34:
 
 impl<T: CoerceUnsized<U>, U> CoerceUnsized<Wrapper<U>> for Wrapper<T> {}
 impl<T: DispatchFromDyn<U>, U> DispatchFromDyn<Wrapper<U>> for Wrapper<T> {}
 
 trait Trait {
 trait Trait {
     // This method isn't object-safe yet. Unsized by-value `self` is object-safe (but not callable
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 2:
 struct Foo<T: ?Sized> {
     a: u16,
-    b: T
+    b: T,
+    b: T,
 }
 
 trait Bar {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 10:
 
 impl Bar for usize {
-    fn get(&self) -> usize { *self }
+    fn get(&self) -> usize {
+    fn get(&self) -> usize {
+        *self
+    }
 }
 
 struct Baz<T: ?Sized> {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 17:
-    a: T
+    a: T,
 
 
 struct HasDrop<T: ?Sized> {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 21:
     ptr: Box<usize>,
+    data: T,
 }
 
 fn main() {
 fn main() {
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 26:
     // Test that zero-offset works properly
-    let b : Baz<usize> = Baz { a: 7 };
+    let b: Baz<usize> = Baz { a: 7 };
     assert_eq!(b.a.get(), 7);
-    let b : &Baz<dyn Bar> = &b;
+    let b: &Baz<dyn Bar> = &b;
     assert_eq!(b.a.get(), 7);
     // Test that the field is aligned properly
     // Test that the field is aligned properly
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 33:
-    let f : Foo<usize> = Foo { a: 0, b: 11 };
+    let f: Foo<usize> = Foo { a: 0, b: 11 };
     assert_eq!(f.b.get(), 11);
-    let ptr1 : *const u8 = &f.b as *const _ as *const u8;
+    let ptr1: *const u8 = &f.b as *const _ as *const u8;
 
-    let f : &Foo<dyn Bar> = &f;
-    let ptr2 : *const u8 = &f.b as *const _ as *const u8;
+    let f: &Foo<dyn Bar> = &f;
+    let ptr2: *const u8 = &f.b as *const _ as *const u8;
     assert_eq!(f.b.get(), 11);
     // The pointers should be the same
     // The pointers should be the same
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 42:
     assert_eq!(ptr1, ptr2);
 
     // Test that nested DSTs work properly
-    let f : Foo<Foo<usize>> = Foo { a: 0, b: Foo { a: 1, b: 17 }};
+    let f: Foo<Foo<usize>> = Foo { a: 0, b: Foo { a: 1, b: 17 } };
     assert_eq!(f.b.b.get(), 17);
-    let f : &Foo<Foo<dyn Bar>> = &f;
+    let f: &Foo<Foo<dyn Bar>> = &f;
     assert_eq!(f.b.b.get(), 17);
     // Test that get the pointer via destructuring works
     // Test that get the pointer via destructuring works
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 51:
 
-    let f : Foo<usize> = Foo { a: 0, b: 11 };
-    let f : &Foo<dyn Bar> = &f;
+    let f: Foo<usize> = Foo { a: 0, b: 11 };
+    let f: &Foo<dyn Bar> = &f;
     let &Foo { a: _, b: ref bar } = f;
     assert_eq!(bar.get(), 11);
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/example/dst-field-align.rs at line 57:
     // Make sure that drop flags don't screw things up
 
-    let d : HasDrop<Baz<[i32; 4]>> = HasDrop {
-        ptr: Box::new(0),
-        data: Baz { a: [1,2,3,4] }
-    };
-    assert_eq!([1,2,3,4], d.data.a);
+    let d: HasDrop<Baz<[i32; 4]>> = HasDrop { ptr: Box::new(0), data: Baz { a: [1, 2, 3, 4] } };
+    assert_eq!([1, 2, 3, 4], d.data.a);
 
-    let d : &HasDrop<Baz<[i32]>> = &d;
-    assert_eq!(&[1,2,3,4], &d.data.a);
+    let d: &HasDrop<Baz<[i32]>> = &d;
+    assert_eq!(&[1, 2, 3, 4], &d.data.a);
 
 
##[error]Diff in /checkout/compiler/rustc_codegen_gcc/build_system/src/clone_gcc.rs at line 34:
                 "--out-path" => match args.next() {
                     Some(path) if !path.is_empty() => out_path = Some(path),
                     _ => {
-                        return Err("Expected an argument after `--out-path`, found nothing".into())
+                        return Err("Expected an argument after `--out-path`, found nothing".into());
                 },
                 "--help" => {
                 "--help" => {
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/compiler/rustc_codegen_gcc/build_system/src/rustc_info.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/clean.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/prepare.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/info.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/fmt.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/config.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/rust_tools.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/test.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/utils.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/build.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/main.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/build_sysroot/lib.rs" "/checkout/tests/mir-opt/const_prop/reify_fn_ptr.rs" "/checkout/tests/mir-opt/const_prop/ref_deref.rs" "/checkout/tests/mir-opt/const_prop/slice_len.rs" "/checkout/tests/mir-opt/const_prop/boxes.rs" "/checkout/tests/mir-opt/const_prop/offset_of.rs" "/checkout/tests/mir-opt/const_prop/indirect.rs" "/checkout/tests/mir-opt/const_prop/checked_add.rs" "/checkout/tests/mir-opt/coroutine_drop_cleanup.rs" "/checkout/tests/mir-opt/set_no_discriminant.rs" "/checkout/tests/mir-opt/issue_120925_unsafefncast.rs" "/checkout/tests/mir-opt/remove_fake_borrows.rs" "/checkout/tests/mir-opt/remove_never_const.rs" "/checkout/tests/mir-opt/remove_unneeded_drops.rs" "/checkout/tests/mir-opt/early_otherwise_branch_soundness.rs" "/checkout/tests/mir-opt/lower_slice_len.rs" "/checkout/tests/mir-opt/tail_call_drops.rs" "/checkout/tests/mir-opt/packed_struct_drop_aligned.rs" "/checkout/tests/mir-opt/const_allocation.rs" "/checkout/tests/mir-opt/match_arm_scopes.rs" "/checkout/tests/mir-opt/simplify_locals_removes_unused_discriminant_reads.rs" "/checkout/tests/mir-opt/box_expr.rs" "/checkout/tests/mir-opt/unusual_item_types.rs" "/checkout/tests/mir-opt/remove_storage_markers.rs" "/checkout/tests/mir-opt/issue_76432.rs" "/checkout/tests/mir-opt/nll/named_lifetimes_basic.rs" "/checkout/tests/mir-opt/nll/region_subtyping_basic.rs" "/checkout/tests/mir-opt/coroutine_storage_dead_unwind.rs" "/checkout/tests/mir-opt/coverage/branch_match_arms.rs" "/checkout/tests/mir-opt/coverage/instrument_coverage.rs" "/checkout/tests/mir-opt/coverage/instrument_coverage_cleanup.rs" "/checkout/tests/mir-opt/issue_99325.rs" "/checkout/tests/mir-opt/issue_41697.rs" "/checkout/tests/mir-opt/array_index_is_temporary.rs" "/checkout/tests/mir-opt/unreachable_enum_branching.rs" "/checkout/tests/mir-opt/uninhabited_enum.rs" "/checkout/tests/mir-opt/no_spurious_drop_after_call.rs" "/checkout/tests/mir-opt/issue_101973.rs" "/checkout/tests/mir-opt/gvn_uninhabited.rs" "/checkout/tests/mir-opt/issue_78192.rs" "/checkout/tests/mir-opt/graphviz.rs" "/checkout/tests/mir-opt/simplify_dead_blocks.rs" "/checkout/tests/mir-opt/issue_72181.rs" "/checkout/compiler/rustc_parse/src/parser/mut_visit/tests.rs" "/checkout/compiler/rustc_parse/src/parser/attr_wrapper.rs" "/checkout/compiler/rustc_parse/src/parser/nonterminal.rs" "/checkout/compiler/rustc_parse/src/parser/ty.rs" "/checkout/compiler/rustc_parse/src/parser/generics.rs" "/checkout/compiler/rustc_parse/src/parser/pat.rs" "/checkout/compiler/rustc_parse/src/parser/expr.rs" "/checkout/compiler/rustc_parse/src/parser/attr.rs" "/checkout/compiler/rustc_parse/src/parser/stmt.rs" "/checkout/compiler/rustc_codegen_gcc/build_system/src/clone_gcc.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Fri Jul 12 12:32:13 UTC 2024
  network time: Fri, 12 Jul 2024 12:32:13 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_codegen_gcc/.rustfmt.toml
+++ b/compiler/rustc_codegen_gcc/.rustfmt.toml
@@ -1,1 +1,3 @@
+version = "Two"
 use_small_heuristics = "Max"
+merge_derives = false

--- a/rustfmt.toml
+++ b/rustfmt.toml
@@ -52,4 +52,5 @@ ignore = [
     "compiler/rustc_codegen_gcc/tests",
     # Code automatically generated and included.
     "compiler/rustc_codegen_gcc/src/intrinsic/archs.rs",
+    "compiler/rustc_codegen_gcc/example",
 ]

--- a/rustfmt.toml
+++ b/rustfmt.toml
@@ -31,7 +31,6 @@ ignore = [
     "library/backtrace",
     "library/portable-simd",
     "library/stdarch",
-    "compiler/rustc_codegen_gcc",
     "src/doc/book",
     "src/doc/edition-guide",
     "src/doc/embedded-book",
@@ -50,4 +49,7 @@ ignore = [
     # These are ignored by a standard cargo fmt run.
     "compiler/rustc_codegen_cranelift/scripts",
     "compiler/rustc_codegen_cranelift/example/gen_block_iterate.rs", # uses edition 2024
+    "compiler/rustc_codegen_gcc/tests",
+    # Code automatically generated and included.
+    "compiler/rustc_codegen_gcc/src/intrinsic/archs.rs",
 ]

